crates/interpreter/src/exec.rs

```diff
@@ -114,9 +115,10 @@ impl<'m> Store<'m> {
     /// The memory is not dynamically allocated and must thus be provided. It is not necessary for
     /// the memory length to be a multiple of 64kB. Execution will trap if the module tries to
     /// access part of the memory that does not exist.
-    pub fn instantiate(
-        &mut self, module: Module<'m>, memory: &'m mut [u8],
-    ) -> Result<InstId, Error> {
+    pub fn instantiate<'a>(
+        &'a mut self, module: Module<'m>, memory: &'m mut [u8],
+    ) -> Result<InstId, Error>
+    where 'a: 'm {
```

Comment:
This is the problem. The actual bound should be the opposite, but we shouldn't need to specify it.

Reply:
It and the one below were suggested by the compiler...